aiida_common_workflows/workflows/relax/vasp/workchain.py

```diff
@@ -30,11 +32,10 @@ def get_forces(forces):
 
 
 @calcfunction
-def get_total_extrapolated_energy(energies):
-    """Return the total extrapolated energy from the energies array."""
-    # We extract by default the energies of the last electronic step, unless special flags are set.
-    total_extrapolated_energy = energies.get_array('energy_extrapolated')[0]
-    total_energy = orm.Float(total_extrapolated_energy)
+def get_total_free_energy(energies):
+    """Return the total free energy from the energies array."""
+    total_free_energy = energies.get_array('energy_free_electronic')[0]
+    total_energy = orm.Float(total_free_energy)
```

Comment:
I see you mix the use of the `DataFactory` with e.g. `orm.Float`. I think it's good to be consistent, and personally really prefer importing `orm` and getting the data types from there because the code looks cleaner and you get tab-completion and proper referencing.

Reply:
Totally agree. I prefer `DataFactory`, since that opens for plugins etc. to use the same interface. But consistency we should fix.